konduit-serving-python/src/main/java/ai/konduit/serving/executioner/PythonExecutioner.java

```diff
@@ -788,6 +807,10 @@ public static PythonVariables execAndReturnAllVariables(String code, PythonVaria
         String inputCode = inputCode(pyInputs);
         return execAndReturnAllVariables(inputCode + code);
     }
+    public static PythonVariables execWithSetupRunAndReturnAllVariables(String code, PythonVariables pyInputs) throws Exception {
```

Comment:
Given all the different static methods here, I wonder if we should just have a builder object and flags instead? @maxpumperla ?

Reply:
@agibsonccc in Python this will translate into a single method with default args. In the end users will only use this through `PythonTransform`, i.e. eventually in `PythonStep`. So I wouldn't worry about this.